Add sample_openapi/README.md (#443)
oxidecomputer/progenitor: sample_openapi/README.md

# Sources

> This directory contains some OpenAPI definitions developed in this project
> solely for the purpose of testing various progenitor features, and
> the following OpenAPI definitions from other projects:

Reviewer: ```suggestion
This directory contains some OpenAPI definitions included in this project
solely for the purpose of testing various progenitor features. Some of these
have been pulled in from other projects:
```

- `api.github.com.json`: https://github.com/github/rest-api-description
- `buildomat.json`: https://github.com/oxidecomputer/buildomat
- `keeper.json`: https://github.com/jclulow/keeper
- `nexus.json`: https://github.com/oxidecomputer/omicron

> - `propolis-server.json`: https://github.com/oxidecomputer/propolis

Reviewer: ```suggestion
- `propolis-server.json`: https://github.com/oxidecomputer/propolis

Note that keeper and buildomat have diverged in order to validate support for various features. The others should not be changed other than updating them from their source repository.
```
